scripts/publish_draft_releases.sh

```diff
@@ -0,0 +1,235 @@
+}
+
+function find_repos() {
+  local teams_json
+  teams_json="${1}"
+
+  org=$(echo "${teams_json}" | jq --raw-output '.org')
+  team_slug=$(echo "${teams_json}" | jq --raw-output '.slug')
+
+  # Use jq --slurp 'add' because pagination can return multiple arrays and we want to treat them as a single array
+  gh api \
+    --paginate \
+    "orgs/${org}/teams/${team_slug}/repos" \
+      | jq \
+          --compact-output \
+          --slurp \
+          'add | map(select(.role_name=="admin")) | map({"full_name":.full_name})'
+}
+
+function publish_draft_release() {
+  local repo_json
+  repo_json="${1}"
+
+  full_name=$(echo "${repo_json}" | jq --raw-output '.full_name')
+  util::print::info "Getting draft releases for ${full_name}"
+
+  # Use jq --slurp 'add' because pagination can return multiple arrays and we want to treat them as a single array
+  draft_releases_json=$(
+    gh api \
+        --paginate \
+        "/repos/${full_name}/releases" \
+          | jq \
+            --compact-output \
+            --slurp \
+            'add | map(select(.draft==true))|map({"id":.id,"tag_name":.tag_name,"target_commitish":.target_commitish})'
+  )
+
+  draft_release_count=$(echo "${draft_releases_json}" | jq length)
+  if [[ "${draft_release_count}" -eq 0 ]]; then
+    util::print::info "↳ None found"
+    util::print::break
+    return
+  fi
+
+  if [[ "${draft_release_count}" -gt 1 ]]; then
+    util::print::error "multiple draft releases found for ${full_name}"
+  fi
+
+  latest_release_json=$(gh api "repos/${full_name}/releases/latest" | jq --compact-output '.')
+  latest_release_tag=$(echo "${latest_release_json}" | jq --raw-output '.tag_name')
+
+  draft_release_id=$(echo "${draft_releases_json}" | jq '.[0].id')
+  draft_release_tag=$(echo "${draft_releases_json}" | jq --raw-output '.[0].tag_name')
+  draft_release_commitish=$(echo "${draft_releases_json}" | jq --raw-output '.[0].target_commitish')
+
+  util::print::title "Draft release found for ${full_name} ($draft_release_tag)"
+  util::print::info "Commits since previous published release (${latest_release_tag}):"
+
+  # Use jq --slurp 'add' because pagination can return multiple arrays and we want to treat them as a single array
+  # Use .[]? to return early (without error) from jq if there are no commits.
+  gh api \
+    --paginate \
+    "/repos/${full_name}/compare/${latest_release_tag}...${draft_release_commitish}" \
+      | jq \
+        --slurp \
+        'add | .commits | .[]? | .commit | {"author":.author.name,"message":.message}'
+
+  # Wait for user input
+  read -p "Publish release for ${full_name} (y/n)? " resp < /dev/tty
+
+  if [[ "${resp}" == "y" || "${resp}" == "yes" ]];then
+    url=$(
+      gh api \
+        --method PATCH \
+        -F draft=false \
+        "/repos/${full_name}/releases/${draft_release_id}" \
+          | jq --raw-output .html_url
+    )
+    util::print::success "release published at: ${url}"
```

Comment:
Likewise, `util::print::success` also exits.

```suggestion
    util::print::green "release published at: ${url}"
```